languages/go/exercises/concept/time-duration/.docs/after.md

```diff
@@ -0,0 +1,14 @@
+A [`Time`][time] in Go is a type describing a moment in time. The date and time information can be accessed, compared, and manipulated through its methods, we have read earlier about some functions called on the `time` package itself. In this exercise we are going to learn about [`Duration`][duration] package.
```

Comment:
> In this exercise we are going to learn 

The `after.md` document should be consider separate from the exercise, as in we'll have students access this document at any point in time, including later on when they'll likely have forgotten the exact context of the exercise. As such, the after.md document should not refer to the exercise.